test/modules/parser/synchronizers/xviz-stream-buffer.spec.js

```diff
@@ -136,10 +143,14 @@ test('XVIZStreamBuffer#insert, getStreams', t => {
   });
 
   const ts1001 = timeslices.find(timeslice => timeslice.timestamp === 1001);
-  t.deepEquals(ts1001.streams, {A: 1.1, C: 1}, 'streams are deep merged');
```

Comment:
From a testing perspective it would be easier to understand if we had a set of tests that explicitly mirrored the table were given, instead of sort of implicitly tested each part of that through the sequence of actions.